### ISSUE TITLE
Green Plum Migration Missing Users?

### ISSUE THREAD

**Issue by rwinch** [spring-attic/pivotal-cla#146]
See https://github.com/pivotalsoftware/pivotal-cla/issues/140#issuecomment-263255735 for the original report.

I looked into this and the individual icla_migration provided from the green plum team did not contain the username of `andreasscherbaum`.  @schubert can you please look into this?

cc @ascherbaum-pivotal @andreasscherbaum

**Comment by ascherbaum-pivotal** [comment]
I'm pretty sure I signed it ;-)

Just some examples:
https://github.com/greenplum-db/gpdb/pull/1193
https://github.com/greenplum-db/gpdb/pull/1101
https://github.com/greenplum-db/gpdb/pull/1057
https://github.com/greenplum-db/gpdb/pull/1050

**Comment by rwinch** [comment]
@ascherbaum-pivotal I think it was a bug in the migration of the original signature. 

@schubert & I will need to coordinate to figure out went wrong and resolve the issue

**Comment by schubert** [comment]
@ascherbaum-pivotal @rwinch yeah I found the thread and the details. We've got the email record of it it but the ICLA PDF was missing from the google drive so it didn't get re-added to the spreadsheet. That's probably my fault.

@ascherbaum-pivotal can you re-sign it? I'll see if there are other users missing.

**Comment by ascherbaum-pivotal** [comment]
@schubert Seriously? Not on file? ;-)

I bounced the message again, to the original address. Please let me know if that works.

**Comment by rwinch** [comment]
Did you guys get this sorted out? Just making sure you don't need any help from my side.

**Comment by schubert** [comment]
@rwinch nothing to do here. @ascherbaum-pivotal the issue is that it just didn't get copied in during the migration but it was on file. 

Can you just click-through the CLA tool and "re-sign" it there?
